frontend/messages/fr.json

```
@@ -4154,5 +4154,53 @@
+	"secIntelFeeds": "Flux de vulnérabilités",
+	"secIntelFeedsDescription": "Configurez des flux de données externes pour l’enrichissement des CVE. Tous les flux sont désactivés par défaut. Activez uniquement ceux dont vous avez besoin — l’application fonctionne entièrement hors ligne sans eux.",
+	"secIntelFeedsSettingsUpdated": "Paramètres des flux de vulnérabilités mis à jour",
+	"kevFeedEnabled": "Activer le flux KEV",
+	"kevFeedEnabledHelpText": "Synchronise quotidiennement le catalogue des vulnérabilités activement exploitées (KEV) de la CISA. Marque les CVE correspondants comme activement exploités (~80 Ko téléchargés).",
+	"epssFeedEnabled": "Activer le flux EPSS",
+	"epssFeedEnabledHelpText": "Synchronise quotidiennement les scores de probabilité d’exploitation EPSS. Met à jour les scores des CVE existants (~15 Mo téléchargés).",
+	"nvdEnrichEnabled": "Activer l’enrichissement automatique NVD",
+	"nvdEnrichEnabledHelpText": "Récupère automatiquement les détails des CVE (CVSS, description, dates) depuis le NVD lors de la création d’un nouveau CVE. Une seule requête par CVE.",
+	"networkTimeout": "Délai d’expiration réseau (secondes)",
+	"networkTimeoutHelpText": "Temps maximal d’attente pour les réponses des flux externes",
+	"securityAdvisories": "Avis de sécurité",
```

Biome (2.4.13)

[error] The key securityAdvisories was already declared.

(lint/suspicious/noDuplicateObjectKeys)